lte/gateway/c/oai/tasks/sgw_s8/sgw_s8_task.c

```diff
@@ -58,7 +58,8 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
   assert(msg_frame);
   MessageDef* received_message_p = (MessageDef*) zframe_data(msg_frame);
```

Comment:
Please use the `receive_msg` function as other tasks are doing since https://github.com/magma/magma/pull/5262/files#diff-0127e0adc6ca4891c8ec0d0ed0f4cc28715b4f7f894c917bbef5faab9ed1965c

Reply:
Thanks a lot Shruti for noticing and guiding